crypto/dsa/dsa_ossl.c

```diff
@@ -189,17 +198,16 @@ static int dsa_sign_setup(DSA *dsa, BN_CTX *ctx_in,
 
     /*
      * We do not want timing information to leak the length of k, so we
-     * compute g^k using an equivalent exponent of fixed length. (This
-     * is a kludge that we need because the BN_mod_exp_mont() does not
-     * let us specify the desired timing behaviour.)
+     * compute G*k using an equivalent scalar of fixed bit-length.
```

Comment:
It's power, not multiplication. I mean `G*k` is misleading here. If you find `^` misleading as well [since it can be viewed as xor], then write `G**k`...

Reply:
Memories of my FORTRAN days :)
I've switch to `^` which matches the comment in `dsa_ossl.c` imediately above.